coreax/metrics.py

```diff
@@ -91,8 +88,8 @@ def compute(
         reference_data: _Data,
         comparison_data: _Data,
         *,
-        block_size: int | None | tuple[int | None, int | None] = None,
-        unroll: tuple[int | bool, int | bool] = (1, 1),
+        block_size: Union[int, None, tuple[Union[int, None], Union[int, None]]] = None,
+        unroll: tuple[Union[int, bool], Union[int, bool]] = (1, 1),
```

Comment:
The type hint and the default here should have been updated in some previous commit to match that of `Kernel.compute_mean`, which is `Union[int, bool, tuple[Union[int, bool], Union[int, bool]]] = 1`. Please can you update to match here

Reply:
Fixed